Aptoide/appcoins-wallet-android
# [APPC-1432] Configure and prepare project to replace fabric by flurry

**What does this PR do?**

   This PR aims to replace Fabric by Flurry in crashlytics

**Database changed?**

No

**Where should the reviewer start?**

- [ ] App.java

**How should this be manually tested?**
Force an error in the app and check if it appears in Flurry

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1432](https://aptoide.atlassian.net/browse/APPC-1432)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1432](https://aptoide.atlassian.net/browse/APPC-1432)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
